docs/using-the-compiler.rst

```diff
@@ -222,14 +231,13 @@ Output Description
                     ]
                   }
                 }
-              }
+              },
               // The same layout as above.
               deployedBytecode: { },
-              // Opcodes list (string)
-              opcodes: "",
               // The list of function hashes
               methodIdentifiers: {
-                "5c19a95c": "delegate(address)",
+                "delegate(address)": "5c19a95c",
```

Comment:
It doesn't matter which order this is in, but this order matches that of `gasEstimates`.